int16 x
int16 y
int16 z
uint16 id1
uint16 id2
uint16 id3
uint16 id4
uint8 pwm1
uint8 motor1
uint8 motor2

